suite-native/atoms/src/Button/IconButton.tsx

```diff
@@ -70,16 +71,23 @@ export const IconButton = ({
 }: IconButtonProps) => {
     const { applyStyle } = useNativeStyles();
 
+    const getIconColor = () => {
```

Comment:
kick this out of component

Reply:
https://github.com/trezor/trezor-suite/pull/7178/commits/35bd1d327d3c7cead98f2ffdb95111f887c7ee68